Gemfile

```diff
@@ -1,9 +1,7 @@
 source 'https://rubygems.org'
 
-# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '5.1.6'
-# See https://qiita.com/shinichinomura/items/41e03d7e4fa56841e654
-gem 'json', '~> 1.8.6'
+gem 'rails', '5.2'
+gem 'json', '~> 1.8'
```

Comment:
__[RuboCop]__

Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `json` should appear before `rails`.


<!-- Comment by [Sider](https://sider.review) -->